transformer_engine/common/gemm/cublaslt_gemm.cu

```diff
@@ -34,6 +35,16 @@ cudaDataType_t get_cuda_dtype(const transformer_engine::DType t) {
   }
 }
 
+uint32_t _getAlignment(uintptr_t address) {
```

Comment:
nit: You don't need to start this function with `_` - it is already with unnamed namespace and not exposed outside this file.